hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -712,6 +716,44 @@ public boolean getReplicationManagerStatus() throws IOException {
 
   }
 
+  @Override
+  public void startContainerBalancer(double threshold, int idleiterations,
+         int maxDatanodesToBalance, long maxSizeToMove) throws IOException {
+
+    StartContainerBalancerRequestProto request =
+        StartContainerBalancerRequestProto.newBuilder()
+            .setIdleiterations(idleiterations)
+            .setMaxDatanodesToBalance(maxDatanodesToBalance)
+            .setMaxSizeToMove(maxSizeToMove)
+            .setThreshold(threshold).build();
+    submitRequest(Type.StartContainerBalancer,
+        builder -> builder.setStartContainerBalancerRequest(request));
+
+  }
+
+  @Override
+  public void stopContainerBalancer() throws IOException {
+
```

Comment:
Unnecessary blank line can be removed.

Reply:
thanks , will remove it

Reply:
Can we remove these extra head and tail black line?